apps/api/src/app/topics/topics.controller.ts

```diff
@@ -53,7 +53,7 @@ import { UserSession } from '../shared/framework/user.decorator';
 import { AnalyticsService } from '../shared/services/analytics/analytics.service';
 import { ANALYTICS_SERVICE } from '../shared/shared.module';
 
-@Controller('topics')
+@Controller('/topics')
```

Comment:
does it not recognize the endpoints without it?

Reply:
I honestly have no idea. But I saw that Subscribers API shows up with that and is the one showing in docs, so I did this guess.